composer.json

```diff
@@ -12,13 +12,13 @@
         { "name": "Jeremy Mikola", "email": "jmikola@gmail.com" }
     ],
     "require": {
-        "php": ">=5.5.0",
+        "php": ">=5.6.0",
         "symfony/framework-bundle": "~2.7|~3.0",
         "symfony/console": "~2.7|~3.0",
         "symfony/form": "~2.7|~3.0",
         "symfony/property-access": "~2.7|~3.0",
         "psr/log": "~1.0",
-        "ruflin/elastica": "3.2.*"
+        "ruflin/Elastica": "~5.0@beta"
```

Comment:
package name should stay lowercase.

and it would be great to keep support for older versions of elastica if possible for people using older ES versions